app/controllers/hosts_controller.rb

```diff
@@ -560,6 +560,7 @@ def process_hostgroup
               Host.new(params[:host])
             end
     @host.set_hostgroup_defaults
+    @host.set_compute_attributes
```

Comment:
Should this only be called if no compute_profile_id is passed via params (i.e. it's inherited)?  Changing the hostgroup when not inheriting the compute profile currently clears the attributes.